Calibration/HcalAlCaRecoProducers/plugins/AlCaHcalIsotrkProducer.cc

```diff
@@ -166,9 +166,9 @@ class AlCaHcalIsotrkProducer : public edm::stream::EDProducer<edm::GlobalCache<a
   const double hitEthrEB_, hitEthrEE0_, hitEthrEE1_;
   const double hitEthrEE2_, hitEthrEE3_;
   const double hitEthrEELo_, hitEthrEEHi_;
-  const edm::InputTag triggerEvent_, theTriggerResultsLabel_;
-  const std::string labelGenTrack_, labelRecVtx_, labelEB_;
-  const std::string labelEE_, labelHBHE_, labelTower_, l1TrigName_;
+  const edm::InputTag triggerEvent_, theTriggerResultsLabel_, labelEB_, labelEE_;
```

Comment:
I personally don't like that some labels are passed as `string`, and some others as `InputTag`... but ok, I think this is a really minor point

Reply:
While ensuring wf 1000.0 works with the default InputTags, I took the liberty to address this in https://github.com/cms-sw/cmssw/pull/47970/commits/d28eba27c329fc22b95532431a34dba88c5d7127